geoips/interfaces/base.py

```diff
@@ -466,19 +466,19 @@ def _create_unvalidated_plugins_cache(self, yaml_plugins):
                 for yaml_subplg in plg_list["spec"][self.name]:
                     try:
                         subplg_names = self._create_plugin_cache_names(yaml_subplg)
+                        for subplg_name in subplg_names:
```

Comment:
This just allows it to skip bad ones, and continue on with the good ones? (to avoid catastrophic failure for a single bad plugin)?

Reply:
Evan: "Yes, when it was outside of the try/except statement, it failed because subplg_names didn't even exist sometimes, yet it was referenced later on in the code. Just ensures that if there are bad plugins we skip, and if they are good, they're processed as usual."